vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -53,10 +52,8 @@ AppTheme::AppTheme(
   });
 }
 
-AppTheme::~AppTheme() = default;
```

Comment:
Why is it removed? There are some advantages keeping default destructors in .cpp file. Especially if the default constructor has to clean up a lot of variables.

Reply:
Could you elaborate a bit?